common/src/Assets/EntityModel.cpp

```diff
@@ -47,10 +47,11 @@ namespace TrenchBroom {
 
         // EntityModel::LoadedFrame
 
-        EntityModelLoadedFrame::EntityModelLoadedFrame(const size_t index, const std::string& name, const vm::bbox3f& bounds) :
+        EntityModelLoadedFrame::EntityModelLoadedFrame(const size_t index, const std::string& name, const vm::bbox3f& bounds, const PitchType pitchType) :
```

Comment:
I wonder if it is correct to set this pitch type by the model type, i.e. MDL gets the different pitch type. Isn't this really dependent of the engine and not the model format? Maybe there are other engines that can read MDL but use regular Euler angles? In that case it would depend on the game configuration.

Reply:
It's MDL specific, but I asked Spike how FTE treats it since it supports many games/formats. There is a cvar "r_meshpitch 1" which disables the pitch inversion for MDL only (contrary to the cvar name), so if you're making a new game in FTE you can opt out of that.

So yes this could potentially have a control in the game config to opt out.